doc/source/serve/advanced-guides/inplace-updates.md

```diff
@@ -111,13 +115,16 @@ The price has updated! The same request now returns `10` instead of `6`, reflect
 
 ## Code Updates
 
-Similarly, you can update any other setting in any deployment in the config file. If a deployment setting other than `num_replicas`, `autoscaling_config`, or `user_config` is changed, it is considered a code update, and the deployment replicas will be restarted. Note that the following modifications are all considered "changes", and will trigger tear down of replicas:
+Changing the following values in a config is considered a code update, and the deployment replicas will be restarted:
+- `ray_actor_options` (per deployment)
+- `import_path` (per application, so all deployments will be restarted)
+- `runtime_env` (per application, so all deployments will be restarted)
```

Comment:
@edoakes Do we want to include the placement group options here?

Reply:
Yes please!